source/server-discovery-and-monitoring/tests/unified/serverMonitoringMode.json

```diff
@@ -0,0 +1,129 @@
+{
+  "description": "serverMonitoringMode",
+  "schemaVersion": "1.3",
+  "tests": [
+    {
+      "description": "connect with serverMonitoringMode=auto",
+      "operations": [
+        {
+          "name": "createEntities",
+          "object": "testRunner",
+          "arguments": {
+            "entities": [
+              {
+                "client": {
+                  "id": "client0",
+                  "uriOptions": {
+                    "serverMonitoringMode": "auto"
+                  }
+                }
+              },
+              {
+                "database": {
+                  "id": "dbSdamModeAuto",
```

Comment:
Nit

dbSdamModeAuto => dbServerMonitoringModeAuto

in all places

Reply:
Done.